### ISSUE TITLE
Feature Request

### ISSUE THREAD

**Issue by rfalk347** [gophish/gophish#726]
We are trying to build a email template with a click here link that displays a tooltip that shows the correct or real URL when you hover over the link to the site people are use to going to. we got the html code only to work on its own but when we add it to the tool it changes it and adds the tracker to track all activity but then sends to user to the real site instead of the phished site. can you guys make that happen or is there already a way and we are just not building it correctly?

**Comment by jordan-wright** [comment]
Can you provide the template you're using? I'm afraid I'm still not quite sure what you're trying to do, but I'll be happy to take a look.

**Comment by rfalk347** [comment]
Here is the html code only we got to work the way we want it to work in GOPHISH but we cant seem to get it work in the gophish tool.

Thanks
Ron

**Comment by rfalk347** [comment]
Sorry about that…..

**Comment by rfalk347** [comment]
Did you get the files i sent over?

**Comment by rfalk347** [comment]
did you get the files I sent over for review

**Comment by jordan-wright** [comment]
No, I don't see any files. Please upload them to this issue and we can continue troubleshooting.

**Comment by rfalk347** [comment]
Here are the files. let me know if you need anything else.



[GOPHISH.ZIP](https://github.com/gophish/gophish/files/1242598/GOPHISH.ZIP)

**Comment by rfalk347** [comment]
I can get the html file to do what I want on its own, but when I try to get the tool to do what the html file does we cant get it to work.

**Comment by rfalk347** [comment]
I added the files.

Thanks
Ron

**Comment by jordan-wright** [comment]
I have the files, but I'm still not quite sure what it is you're trying to do. You said:

> We are trying to build a email template with a click here link that displays a tooltip that shows the correct or real URL when you hover over the link to the site people are use to going to.

You mean that the tooltip appears in the email client? I don't see where in the HTML you're doing that. I see a reference to a `false=` attribute, but I don't see that as part of the spec for the `a` tag.

Can you please explain what you're trying to do, what in the HTML you've done that you think will accomplish that, and what specifically Gophish is doing to the HTML so that this isn't working?

**Comment by rfalk347** [comment]
The 1adp gophish email template source code.html file standalone works as we want it to. In the tool  however If we copy the code into the tool as the email template and save the template it does not take them to the landing page we setup in the adp final landing page3.html it actually takes them to the real url which we don’t want. Looks like once the code is in the tool the tool changes it to its own and wont work like the 1adp gophish email template source code.html file by itself if you click on the file and run.

Thanks
Ron

**Comment by rfalk347** [comment]
Any luck on this??

thanks

From: Falk, Ron (TFG)
Sent: Wednesday, August 23, 2017 9:20 AM
To: gophish/gophish <reply@reply.github.com>
Subject: RE: [gophish/gophish] Feature Request (#726)

The 1adp gophish email template source code.html file standalone works as we want it to. In the tool  however If we copy the code into the tool as the email template and save the template it does not take them to the landing page we setup in the adp final landing page3.html it actually takes them to the real url which we don’t want. Looks like once the code is in the tool the tool changes it to its own and wont work like the 1adp gophish email template source code.html file by itself if you click on the file and run.

Thanks
Ron

**Comment by jordan-wright** [comment]
I'm afraid I haven't had time to look into this yet. I'll update the issue when I have an update.

Thanks!

**Comment by rfalk347** [comment]
Thank you Sir….
